webrtc/RTCPeerConnection-ontrack.html

```diff
@@ -0,0 +1,49 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>RTCPeerConnection.prototype.ontrack</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+  'use strict';
+
+  // tests that ontrack is called and parses the msid information from the SDP and creates
+  // the streams with matching identifiers.
+  async_test(t => {
+    const pc = new RTCPeerConnection();
+
+    // Fail the test if the ontrack event handler is not implemented
+    assert_own_property(pc, 'ontrack', 'Expect pc to have ontrack event handler attribute');
+
+    const sdp = `v=0
+o=- 166855176514521964 2 IN IP4 127.0.0.1
+s=-
+t=0 0
+a=msid-semantic:WMS *
+m=audio 9 UDP/TLS/RTP/SAVPF 111
+c=IN IP4 0.0.0.0
+a=rtcp:9 IN IP4 0.0.0.0
+a=ice-ufrag:someufrag
+a=ice-pwd:somelongpwdwithenoughrandomness
+a=fingerprint:sha-256 8C:71:B3:8D:A5:38:FD:8F:A4:2E:A2:65:6C:86:52:BC:E0:6E:94:F2:9F:7C:4D:B5:DF:AF:AA:6F:44:90:8D:F4
+a=setup:actpass
+a=rtcp-mux
+a=mid:mid1
+a=sendonly
+a=rtpmap:111 opus/48000/2
+a=msid:stream1 track1
+a=ssrc:1001 cname:some
+`;
+
+    pc.ontrack = t.step_func(event => {
+      assert_equals(event.streams.length, 1, 'the track belongs to one MediaStream');
+      assert_equals(event.streams[0].id, 'stream1', 'the stream name is parsed from the MSID line');
+      t.done();
+    });
+
+    pc.setRemoteDescription(new RTCSessionDescription({type: 'offer', sdp: sdp}))
+    .catch(t.step_func(err => {
+      assert_unreached('Error ' + err.name + ': ' + err.message);
+    }));
+  }, 'setRemoteDescription should trigger ontrack event when the MSID of the stream is is parsed.');
```

Comment:
I haven't really read through in detail but this is one of the old tests written by @fippo. Currently just moving it as is to a more appropriate `ontrack` test file. Planning to study and write detailed tests for `ontrack` soon in another PR.